apps/android/app/src/main/AndroidManifest.xml

```diff
@@ -38,7 +38,6 @@
         android:windowSoftInputMode="adjustResize"
         android:theme="@style/Theme.SimpleX">
       <intent-filter>
-        <action android:name="android.intent.action.MAIN" />
```

Comment:
why is it removed? I think something might depend on it, don't remember exactly

Reply:
Activity-aliases have this action. They receive it